README.md

```diff
@@ -70,6 +70,8 @@ The language itself draws pieces of influence from:
 
 # Contributors
 
+Information and instructions for potential contributors are provided [here](CONTRIBUTING).
```

Comment:
```suggestion
Information and instructions for potential contributors are provided [here](.github/CONTRIBUTING.md).
```

Reply:
Moving .github/CONTRIBUTING to the root folder.